sycl/include/sycl/detail/pi.h

```diff
@@ -127,6 +131,11 @@ using pi_bool = pi_uint32;
 using pi_bitfield = pi_uint64;
 using pi_native_handle = uintptr_t;
 
+typedef struct _ze_command_queue_handle_t *ze_command_queue_handle_t;
+typedef struct _ze_command_list_handle_t *ze_command_list_handle_t;
+using pi_native_handle2 =
+    std::variant<ze_command_queue_handle_t, ze_command_list_handle_t>;
```

Comment:
Can we avoid mentioning backend specific types here? Cannot find that they are used in this file?